pkg/controller/kubernetes/features/kube_features.go

```diff
@@ -0,0 +1,697 @@
+/*
```

Comment:
please remove `kube_features.go` file, and define the feature gate as true defaultly.